composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "ext-pdo": "*"
```

Comment:
Since the test case does not use the trait, I wonder if adding this is correct. Should it not be added in the composer.json file of the bundle that uses the trait? Because `contao/calendar-bundle` currently has no functional tests but still requires the test case; and it works perfectly fine without `ext-pdo`.

Reply:
🤔 valid question, I don't know how this is handled. Maybe it should be suggested only? And it should be a dev requirement?